libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm/primarySchoolSection/reasonForApplicationSubSection.ts

```diff
@@ -16,6 +17,11 @@ export const reasonForApplicationSubSection = buildSubSection({
   id: 'reasonForApplicationSubSection',
   title:
     newPrimarySchoolMessages.primarySchool.reasonForApplicationSubSectionTitle,
+  condition: (answers) => {
+    // Only display section if not seleting neighbourhood school
+    const { applyForNeighbourhoodSchool } = getApplicationAnswers(answers)
+    return applyForNeighbourhoodSchool === NO
+  },
```

Comment:
This page should also appear if `NEW_PRIMARY_SCHOOL`.
```suggestion
  condition: (answers) => {
    // Only display section if not seleting neighbourhood school
    const { applicationType, applyForNeighbourhoodSchool } =
      getApplicationAnswers(answers)
    return (
      applicationType === ApplicationType.NEW_PRIMARY_SCHOOL ||
      (applicationType === ApplicationType.ENROLLMENT_IN_PRIMARY_SCHOOL &&
        applyForNeighbourhoodSchool === NO)
    )
  },
```